api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/calculation/BasePatientCalculationTest.java

```diff
@@ -21,88 +23,101 @@
-    evaluationContext = service.createCalculationContext();
-
-    Map<String, Object> cacheEntries = new HashMap<String, Object>();
-    cacheEntries.put("location", Context.getLocationService().getLocation(1));
-    setEvaluationContext(testsHelper.getDate("2019-05-30 00:00:00.0"));
-    setEvaluationContext(cacheEntries);
-
-    executeDataSet("calculationsTest.xml");
-
-    PatientCalculation calculation = getCalculation();
-    new EptsCalculationsMock(calculation);
-    new EptsCalculationUtilsMock();
-  }
-
-  protected void setEvaluationContext(Date now) {
-    evaluationContext.setNow(now == null ? new Date() : now);
-  }
-
-  @SuppressWarnings("rawtypes")
-  protected void setEvaluationContext(Map<String, Object> cacheEntries) {
-    if (cacheEntries != null && !cacheEntries.isEmpty()) {
-      for (Map.Entry e : cacheEntries.entrySet()) {
-        evaluationContext.addToCache((String) e.getKey(), e.getValue());
-      }
-    }
-  }
-
-  public PatientCalculationContext getEvaluationContext() {
-    return evaluationContext;
-  }
-
-  protected void matchOtherResultsExcept(
-      CalculationResultMap evaluatedResult, Integer... patientsIdNotToMatch) {
-    CalculationResultMap otherResult = (CalculationResultMap) evaluatedResult.clone();
-    CalculationResultMap initialResult = (CalculationResultMap) getResult().clone();
-    if (patientsIdNotToMatch != null) {
-      for (int i : patientsIdNotToMatch) {
-        initialResult.remove(i);
-        otherResult.remove(i);
-      }
-    }
-    Assert.assertTrue(matchCalculationResults(initialResult, otherResult));
-  }
-
-  protected boolean matchCalculationResults(
-      CalculationResultMap results1, CalculationResultMap results2) {
-    boolean matched = true;
-    if (results1.size() != results2.size() || !results1.keySet().equals(results2.keySet())) {
-      matched = false;
-    } else {
-      Iterator<Integer> results1Iterator = results1.keySet().iterator();
-      while (results1Iterator.hasNext()) {
-        Integer key = results1Iterator.next();
-        if (results1.get(key).getValue() == null) {
-          matched = results2.get(key).getValue() == null;
-        } else if (results2.get(key).getValue() == null) {
-          matched = results1.get(key).getValue() == null;
-        } else if (!results1.get(key).getValue().equals(results2.get(key).getValue())) {
-          matched = false;
-        }
-      }
-    }
-    return matched;
-  }
+	
+	@Autowired
+	protected TestsHelper testsHelper;
+	
+	@Autowired
+	protected OpenMRSTestHelper openmrsTestHelper;
+	
+	protected PatientCalculationService service;
+	
+	private PatientCalculationContext evaluationContext;
+	
+	protected Map<String, Object> params = new HashMap<String, Object>();
+	
+	public abstract PatientCalculation getCalculation();
+	
+	public abstract Collection<Integer> getCohort();
+	
+	public abstract CalculationResultMap getResult();
+	
+	@Before
+	public void setUp() throws Exception {
```

Comment:
am seeing wrong indentations @nthfloor hope u are running mvn clean install to run the formatter. i also hope these 4 last commits from your are not making me loose any of my changes!

Reply:
I think the formatting was my git tool. For some reason it did not register the formatter changes, after refreshing I can still see outstanding changes which are are the formatter.

As for loosing any of your changes. Have a look for yourself, everything is there with the exception of the spring dependency injection adjustments.